docs/source/external_redirects.rst

```diff
@@ -0,0 +1,13 @@
+:orphan:
```

Comment:
I saw there is another "redirects" sphinx extension that also supports redirects to external URLs: https://gitlab.com/documatt/sphinx-reredirects

Just checking: `sphinx-rediraffe` does not support this, right? That would sound like a very useful feature; it seems a bit weird to redirect all the missing content to a page, where one then creates a list of links.

Reply:
Can you open an issue for this?

Reply:
No it doesn't, also maybe it feels like it could be a bit disorientating (even unwanted) for the user to be redirected externally without asking.

Anyhow, I doubt many people will search for these legacy pages

Reply:
Well, usually if I click on a page as a user  (search result, ...)  it is because I'm interested in its content.
If a domain no longer serves this content, I would be happy to be redirected to the place where I can find it